test/logging/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM python
```

Comment:
Please move the base to cortx-prvsnr/experiments folder.

Reply:
Done